site-src/geps/gep-709.md

```diff
@@ -82,8 +88,8 @@ decisions:
    more powerful it may encourage unnecessarily insecure configuration.
 
 ```go
-// ReferencePolicy identifies cross namespace relationships that are trusted for
-// Gateway API.
+// ReferencePolicy identifies cross namespace relationships between types of
```

Comment:
In the previous PR, a suggestion was made to rename this to [InboundReferencePolicy](https://github.com/kubernetes-sigs/gateway-api/pull/711#discussion_r670025962). Wanted to pick up the conversation in this PR. I think I slightly prefer the shorter `ReferencePolicy` name, but admit that `InboundReferencePolicy` would make room for something like `OutboundReferencePolicy`. That might allow admins to restrict where Gateways in their namespace could forward to. Thoughts?

/cc @youngnick @bowei @danehans @hbagdi @jpeach

Reply:
Yeh, my preference is also for a single ReferencePolicy with From+To.

Reply:
Yeah as I've been thinking about it more, it would not be that difficult to expand ReferencePolicy to cover both inbound and outbound references. I don't think that's something we actually want to do, but the only required change would be adding `namespace` to the `to` struct. The larger hurdle would be the complexity for users in trying to restrict/allow both inbound and outbound references.

Reply:
As discussed in community meeting today, we believe it is unlikely that we will ever have a need for an OutboundReferencePolicy. Given that, the preference is to keep the simpler `ReferencePolicy` name. I'll update the comments around ReferencePolicy to make it clearer that it refers to inbound references.

Reply:
I've updated the wording around this, PTAL